### PR TITLE
Product listing page added

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
good to me

**Comment by ishitakeshawani** [comment]
It will be great if you can provide a preview link.

File: css/variable.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:root {
+  --primary-color: #cfaa89;
+  --primary-color-bg: #dbc1aa52;
+
```
Comment (ishitakeshawani) [inline css/variable.css:4]: Please remove extra space.

**Review by ishitakeshawani — APPROVED** [review]
LGTM

File: pages/productList/index.html [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+          </div>
+          <!-- end of category selection -->
+
+          <!-- selection based on rating -->
+          <div class="flex-col gap-sm">
+            <h4>Rating</h4>
+          <label ><input type="radio" name="" id=""> 4 stars and above</label>
+          <label ><input type="radio" name="" id=""> 3 stars and above</label>
+          <label ><input type="radio" name="" id=""> 2 stars and above</label>
+          <label ><input type="radio" name="" id=""> 1 stars and above</label>
```
Comment (Naveenchandar) [inline pages/productList/index.html:136]: if  name and id attributes are not used, you can remove.

File: pages/productList/index.html [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+      </div>
+      <!-- end of filter section -->
+
+      <!-- product-list -->
+      <div class="flex-row-wrap pd-m gap-xl flex-center">
+        <!-- 1st card -->
+        <div class="card-ecom">
+          <span class="card-badge bg-col-prim">#1 Bestseller</span>
+          <button class="like-btn">
+            <img src="/media/icons/redHeart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:164]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+            class="responsive-img"
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Vinaya Beauty</h4>
+            <p class="marg-un">Handmade soap set</p>
+            <strong> Rs. 749 </strong> <s>999</s> <span>25% OFF</span>
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:178]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+          </button>
+        </div>
+
+        <!-- 2nd card -->
+        <div class="card-ecom">
+          <span class="card-badge bg-col-prim">#1 Bestseller</span>
+
+          <button class="like-btn">
+            <img src="/media/icons/redHeart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:187]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Vilas dining</h4>
+            <p class="marg-un">Printed designer kitchen plates</p>
+            <strong> Rs. 1349 </strong> <s>2139</s> <span>35% OFF</span>
+
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:202]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 7]
```diff
@@ -0,0 +1,396 @@
...
+          </button>
+        </div>
+       
+
+        <!-- 3rd card -->
+        <div class="card-ecom">
+          <span class="card-badge bg-col-prim">Trending</span>
+          <button class="like-btn">
+            <img src="/media/icons/redHeart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:211]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 8]
```diff
@@ -0,0 +1,396 @@
...
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Nature jewels</h4>
+            <p class="marg-un">Storage basket</p>
+            <strong> Rs. 1299 </strong> <s>1699</s> <span>23% OFF</span>
+
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:226]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 9]
```diff
@@ -0,0 +1,396 @@
...
+          </button>
+        </div>
+
+        <!-- 4rth card -->
+        <div class="card-ecom">
+          <span class="card-badge bg-col-prim">Trending</span>
+          <button class="like-btn">
+            <img src="/media/icons/redHeart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:234]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 10]
```diff
@@ -0,0 +1,396 @@
...
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Windbox Decor</h4>
+            <p class="marg-un">Set of wall portraits</p>
+            <strong> Rs. 749 </strong> <s>999</s> <span>25% OFF</span>
+
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:249]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 11]
```diff
@@ -0,0 +1,396 @@
...
+          </button>
+        </div>
+       
+
+        <!-- 5th card -->
+        <div class="card-ecom">
+          <!-- <span class="card-badge bg-col-prim">#1 Bestseller</span> -->
+          <button class="like-btn">
+            <img src="/media/icons/redHeart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:258]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 12]
```diff
@@ -0,0 +1,396 @@
...
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Nature jewels</h4>
+            <p class="marg-un">Wooden Handmade toy</p>
+            <strong> Rs. 895 </strong> <s>1489</s> <span>40% OFF</span>
+
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:273]: Please provide alt description for accessibility

File: pages/productList/index.html [hunk 13]
```diff
@@ -0,0 +1,396 @@
...
+          />
+          <div class="product-details">
+            <h4 class="marg-un">Mansa creations</h4>
+            <p class="marg-un top-fit">Ceramic vase mustard colored</p>
+            <strong> Rs. 1456 </strong> <s>1799</s> <span>40% OFF</span>
+
+          </div>
+          <button class="cart-btn">
+            Add to cart
+            <img src="/media/icons/bluecart.svg" />
```
Comment (Naveenchandar) [inline pages/productList/index.html:296]: Please provide alt description for accessibility

File: pages/productList/productList.css [hunk 14]
```diff
@@ -0,0 +1,83 @@
+.nav-head {
+  position: relative;
+}
+
+.parting {
+  justify-content: space-around;
+  justify-content: space-evenly;
```
Comment (Naveenchandar) [inline pages/productList/productList.css:7]: Either one will work. So, you can remove the unused property.

Comment (surbhi84) [reply pages/productList/productList.css:7]: yeah i forgot to remove that

File: pages/productList/index.html (hunk 13 above)
Comment (surbhi84) [reply pages/productList/index.html:296]: added in another branch for all images

**Comment by surbhi84** [comment]
i have made the changes in the child branch of this branch,

**Pull request merged**